### DISCUSSION TITLE
Please use the `v6.0.102-source-build` tag instead of `v6.0.102` for source-build

### DISCUSSION THREAD

**Discussion by omajid** [dotnet/source-build#2756] [General]
The "official" `v6.0.102` tag is broken for source-build purposes.

See https://github.com/dotnet/installer/releases/tag/v6.0.102-source-build

cc @alucryd @asbjornu @ayakael @cho-m @crummel @livius90

**Comment by alucryd** [comment]
@omajid Thanks for the heads up!

Well, my automatic build of `v6.0.102-source-build` is broken as well: https://build.alucryd.xyz/#/builders/257/builds/244/steps/12/logs/stdio

Haven't had time to look into it yet, but the previous release was broken too anyway on Arch, haven't been able to build anything since the initial 6.0 release.

**Comment by ayakael** [comment]
I have this issue on Alpine as well. Xplat build doesn't seem to know to look for source-built-packages. I fixed my APKBUILD by setting /p:PrebuiltPackagesPath=/path/to/prebuilt/packages, thus:


./build.sh \\

\-- \\

/p:PrebuiltPackagesPath="$artifactsdir"

I don't quite remember where Private.SourceBuilt.Artifacts tar is extracted to under $builddir/packages, but it needs to point to that. I think it's about time I open an issue on this.

**Comment by alucryd** [comment]
@ayakael That did the trick, thanks!
